mlflow/server/handlers.py

```diff
@@ -2637,6 +2639,28 @@ def _finalize_logged_model(model_id: str):
     return _wrap_response(response_message)
 
 
+@catch_mlflow_exception
+@_disable_if_artifacts_only
+def _set_logged_model_tags(model_id: str):
+    request_message = _get_request_message(
+        SetLoggedModelTags(),
+        schema={
+            "model_id": [_assert_string, _assert_required],
+            "tags": [_assert_array],
+        },
+    )
+    tags = [LoggedModelTag(key=tag.key, value=tag.value) for tag in request_message.tags]
+    _get_tracking_store().set_logged_model_tags(request_message.model_id, tags)
+    return _wrap_response(SetLoggedModelTags.Response())
+
+
+@catch_mlflow_exception
+@_disable_if_artifacts_only
+def _delete_logged_model_tag(model_id: str, tag_key: str):
+    _get_tracking_store().delete_logged_model_tag(model_id, tag_key)
```

Comment:
Why don't we use _get_request_message and check schema here?

Reply:
Good question. The DeleteLoggedModelTag definition looks like this:

```
  rpc deleteLoggedModelTag(DeleteLoggedModelTag) returns (DeleteLoggedModelTag.Response) {
    option (rpc) = {
      endpoints: [{
        method: "DELETE",
        path: "/mlflow/logged-models/{model_id}/tags/{tag_key}"
        since {major: 2, minor: 0},
      }],
      visibility: PUBLIC_UNDOCUMENTED,
      rpc_doc_title: "Delete Logged Model Tag",
    };
  }
```

`model_id` and `tag_key` are path parameters, not query parameters.

Reply:
@serena-ruan IMO, this is inconsistent and should be refactored.